### PR TITLE
DOCSUP-8026: Document the settings for customizing timeouts

### PR DESCRIPTION
**Pull request by adevyatova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #20088

### CODE REVIEW

File: docs/ru/operations/settings/merge-tree-settings.md [hunk 1]
```diff
@@ -149,6 +149,39 @@ Eсли суммарное число активных кусков во все
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## replicated_fetches_http_send_timeout {#replicated_fetches_http_send_timeout}
+
+Тайм-аут отправки HTTP (в секундах) для запросов на скачивание кусков. Наследуется от профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```
Comment (olgarev) [inline docs/ru/operations/settings/merge-tree-settings.md:165]:
```suggestion
Тайм-аут отправки (в секундах) для HTTP-запросов на скачивание кусков. Наследуется из профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```

File: docs/ru/operations/settings/merge-tree-settings.md [hunk 2]
```diff
@@ -149,6 +149,39 @@ Eсли суммарное число активных кусков во все
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## replicated_fetches_http_receive_timeout {#replicated_fetches_http_receive_timeout}
+
+Тайм-аут приёма HTTP (в секундах) для запросов на скачивание кусков. Наследуется от профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```
Comment (olgarev) [inline docs/ru/operations/settings/merge-tree-settings.md:176]:
```suggestion
Тайм-аут (в секундах) для приёма HTTP-запросов на скачивание кусков. Наследуется из профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```

File: docs/ru/operations/settings/merge-tree-settings.md [hunk 3]
```diff
@@ -149,6 +149,39 @@ Eсли суммарное число активных кусков во все
 
 Стандартное значение Linux dirty_expire_centisecs - 30 секунд (максимальное время, которое записанные данные хранятся только в оперативной памяти), но при больших нагрузках на дисковую систему, данные могут быть записаны намного позже. Экспериментально было найдено время - 480 секунд, за которое гарантированно новый кусок будет записан на диск.
 
+## replicated_fetches_http_connection_timeout {#replicated_fetches_http_connection_timeout}
+
+Тайм-аут HTTP-соединения (в секундах) для запросов на скачивание кусков. Наследуется от профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```
Comment (olgarev) [inline docs/ru/operations/settings/merge-tree-settings.md:154]:
```suggestion
Тайм-аут HTTP-соединения (в секундах) для запросов на скачивание кусков. Наследуется из профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
```

File: docs/ru/operations/settings/settings.md [hunk 4]
```diff
@@ -2857,5 +2857,37 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## http_send_timeout {#replicated_fetches_http_send_timeout}
+
+Тайм-аут отправки HTTP (в секундах).
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2873]:
```suggestion
Тайм-аут для отправки HTTP-запросов (в секундах).
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -2857,5 +2857,37 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
 │ 109 │
 └─────┘
 ```
+## http_connection_timeout {#http_connection_timeout}
+
+Тайм-аут HTTP-соединения (в секундах).
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2862]:
```suggestion
Тайм-аут для HTTP-соединения (в секундах).
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -2857,5 +2857,37 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## http_receive_timeout {#http_receive_timeout}
+
+Тайм-аут приема HTTP (в секундах).
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:2884]:
```suggestion
Тайм-аут для получения HTTP-запроса (в секундах).
```

File: docs/en/operations/settings/settings.md [hunk 7]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
 Default value: `60`.
 
+## http_connection_timeout {#http_connection_timeout}
+
+HTTP connection timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2873]:
```suggestion
-   0 - Disabled (infinite timeout).
```

File: docs/en/operations/settings/settings.md [hunk 8]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+Default value: 0.
+
+## http_send_timeout {#replicated_fetches_http_send_timeout}
+
+HTTP send timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2884]: Same

File: docs/en/operations/settings/settings.md [hunk 9]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+## http_connection_timeout {#http_connection_timeout}
+
+HTTP connection timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
+
+Default value: 0.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2875]:
```suggestion
Default value: 1.
```

File: docs/en/operations/settings/settings.md [hunk 10]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+## http_send_timeout {#replicated_fetches_http_send_timeout}
+
+HTTP send timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
+
+Default value: 0.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2886]:
```suggestion
Default value: 1800.
```

File: docs/en/operations/settings/settings.md [hunk 11]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+## http_receive_timeout {#http_receive_timeout}
+
+HTTP receive timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
+
+Default value: 0.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2897]:
```suggestion
Default value: 1800.
```

File: docs/en/operations/settings/settings.md [hunk 12]
```diff
@@ -2863,6 +2863,39 @@ Sets the interval in seconds after which periodically refreshed [live view](../.
...
+
+## http_receive_timeout {#http_receive_timeout}
+
+HTTP receive timeout (in seconds).
+
+Possible values:
+
+-   Any positive integer.
+-   0 - Disabled.
```
Comment (tavplubix) [inline docs/en/operations/settings/settings.md:2895]: Same

File: docs/ru/operations/settings/settings.md [hunk 13]
```diff
@@ -2857,5 +2857,37 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## http_send_timeout {#replicated_fetches_http_send_timeout}
+
+Тайм-аут для отправки HTTP-запросов (в секундах).
```
Comment (tavplubix) [inline docs/ru/operations/settings/settings.md:2873]:
```suggestion
Тайм-аут для отправки данных через HTTP-интерфейс (в секундах).
```

File: docs/ru/operations/settings/settings.md [hunk 14]
```diff
@@ -2857,5 +2857,37 @@ SELECT * FROM test LIMIT 10 OFFSET 100;
...
+Возможные значения:
+
+-   0 - тайм-аут не задан.
+-   Любое положительное целое число.
+
+Значение по умолчанию: `0`.
+
+## http_receive_timeout {#http_receive_timeout}
+
+Тайм-аут для получения HTTP-запроса (в секундах).
```
Comment (tavplubix) [inline docs/ru/operations/settings/settings.md:2884]:
```suggestion
Тайм-аут для получения данных через HTTP-интерфейс (в секундах).
```

**Comment by adevyatova** [comment]
@tavplubix Review, please, I fixed it!

File: docs/ru/operations/settings/merge-tree-settings.md [hunk 15]
```diff
@@ -149,6 +149,39 @@ Eсли суммарное число активных кусков во все
 
 Стандартное значение Linux dirty_expire_centisecs - 30 секунд (максимальное время, которое записанные данные хранятся только в оперативной памяти), но при больших нагрузках на дисковую систему, данные могут быть записаны намного позже. Экспериментально было найдено время - 480 секунд, за которое гарантированно новый кусок будет записан на диск.
 
+## replicated_fetches_http_connection_timeout {#replicated_fetches_http_connection_timeout}
+
+Тайм-аут HTTP-соединения (в секундах) для запросов на скачивание кусков. Наследуется из профиля по умолчанию [http_connection_timeout](./settings.md#http_connection_timeout), если не задан явно.
+
+Возможные значения:
+
+-   0 - тайм-аут не задан.
```
Comment (tavplubix) [inline docs/ru/operations/settings/merge-tree-settings.md:158]:
```suggestion
-   0 - используется значение `http_connection_timeout`.
```

**Review by tavplubix — APPROVED** [review]

**Pull request merged**